test/envoye2e/stats_plugin/stats_plugin_test.go

```diff
@@ -0,0 +1,208 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_test
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/proxy/test/envoye2e/env"
+)
+
+const outboundStatsFilter = `- name: envoy.filters.http.wasm
```

Comment:
Can we change all these from inline string to just yaml files?
They can be templatized if needed.
These are the authoritative config files, which should be as plain yamls.

Reply:
These are already templatized in envoy_conf.go and tcp_envoy_conf.go.. 
the only part that is inline are extra filters or custom data...

like it this way as it shows in test file itself what extra config we are testing..